src/azul/chalice.py

```diff
@@ -96,6 +98,11 @@ def authentication_middleware(self, event, get_response):
             response = get_response(event)
         return response
 
+    def _lambda_context_middleware(self, event, get_response):
+        if not self.unit_test:
```

Comment:
What would happen if we omitted the conditional?

Reply:
This: https://github.com/DataBiosphere/azul/runs/5633147799?check_suite_focus=true

But I think a better solution is https://github.com/DataBiosphere/azul/pull/3943/commits/ed1a735dbd008d8f129bf70d0db745514a72cb7c